### PR TITLE
 Fix show heating state when away

### LINKED ISSUE #6: When the preset mode is away, heat_demand is ignored
When the preset mode of the controller is set to away and a zone is showing heat demand because the temperature is lower then the setpoint, this isn't visible in the climate entity. 

The state of that entity(zone) should not be "auto" but should be "head", as there is heat demand. 
As for the state of the controller is also showing "heat_demand" > 0 , so the state should also reflect that
[home-assistant.log](https://github.com/zxdavb/evohome_cc/files/5871719/home-assistant.log)

### PR DESCRIPTION
**Pull request by phdelodder**
This should fix #6

### CODE REVIEW

File: custom_components/evohome_cc/climate.py [hunk 1]
```diff
@@ -243,7 +245,10 @@ def hvac_mode(self) -> Optional[str]:
             return
         if self._evo_device.mode["system_mode"] == "heat_off":
             return HVAC_MODE_OFF
-        if self._evo_device.mode["system_mode"] == "away":
+        if self.i_evo_device.mode["system_mode"] == "away":
```
Comment (zxdavb) [inline custom_components/evohome_cc/climate.py:248]: Is `i_evo_device.mode` a typo?

**Comment by zxdavb** [comment]
Have you confused `CURRENT_HVAC_*` with `HVAC_MODE_*`?

File: custom_components/evohome_cc/climate.py (hunk 1 above)
Comment (phdelodder) [reply custom_components/evohome_cc/climate.py:248]: Yes it's a typo

**Comment by phdelodder** [comment]
Rethinking about it

**Pull request closed without merging**